schema/bom-1.6.proto

```diff
@@ -1482,17 +1482,29 @@ message Condition {
 
 enum TaskType {
   TASK_TYPE_COPY = 0;
+  // A task that copies software or data used to accomplish other tasks in the workflow.
```

Comment:
Comments go over the accosted field,
not under them.
please change

Reply:
Sorry for that... 3rd time is a charm ;)

Reply:
LGTM